ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -133,7 +136,22 @@ type opennebula_ceph_datastore = {
     "rbd_format"                ? long(1..2)
 };
 
-@{ type for an opennebula datastore. Defaults to a ceph datastore (ds_mad is ceph) @}
+@documentation{ 
+type for vnet ars specific attributes. 
+type and size are mandatory 
+}
+type opennebula_ar = {
+    "type"                      : string with match(SELF, "^(IP4|IP6|IP4_6|ETHER)$")
+    "ip"                        ? type_ipv4
```

Comment:
Okay I see, it's because their IPv6 support is only for auto-configured addresses.